ui/app/src/modules/Preview/Tools/ContentTree.tsx

```diff
@@ -256,7 +256,7 @@ function getChildren(
 }
 
 interface TreeItemCustomInterface {
-  nodes: RenderTree;
+  nodes: any;
```

Comment:
Incorrect type. Use Resource<RenderTree>

Reply:
I have to set fallback types for this prop because it changes its type within its render. I tried to separate them on two props but by some reason it did not work and crashed the site with no error display.
changed to: 
nodes: Resource<Data> | RenderTree | any;